pkg/relic/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME=relic
-PKG_URL=https://github.com/relic-toolkit/relic.git
-PKG_VERSION=cdcfaeef101d18c3231c3b46359c519dd72682e8
+PKG_URL=https://github.com/tfar/relic.git
```

Comment:
@tfar would you be fine having the relic specific branch under the RIOT organization instead of keeping it in your private Github account? This way, every RIOT maintainer would have write access to it for future changes/fixes,

Reply:
Sure happy with that. Shall I clone it under the RIOT org?